app/views/admin/checks/employment.html.erb

```diff
@@ -9,7 +9,15 @@
           heading: "Employment",
           description: I18n.t("#{@claim.policy.to_s.underscore}.admin.employment_check_description"),
           answers: @eligibility_checks.employment %>
+  </div>
 
-    <%= button_to "Complete employment check and continue", admin_claim_checks_path(@claim, check: :employment), class: "govuk-button", method: :post %>
+  <div class="govuk-grid-column-full">
+    <% if @check.present? %>
+      <p class="govuk-body">
+        Checked by <strong><%= user_details(@check.created_by, include_line_break: false) %></strong> on <strong><%= l(@check.created_at) %></strong>
```

Comment:
I'll double check with @titlescreen but I think it should be ok, the guidance says to use them sparingly to emphasis particular words. I'll extract it out to a partial as well 👍 
https://design-system.service.gov.uk/styles/typography/#bold-text

Reply:
So the `<strong>` tag should allow the screen reader to those words in the same way that a heavier font-weight would visually emphasise the words. So if we want to emphasize these words then that is the way to go.

I take it that it would normally only say the name like `Joe Bloggs` and the sign-in ID would not be present? If that is the case then I think we have used it sparingly enough.

Reply:
Yeah usually it will just display the name without the ID. Originally I tried displaying it in a summary list, but it looked strange next to but disconnected from the summary list above it.